### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by jorivanee**
Move discord import to the beginning of file, aswell as the version check, if the version wasn't atleast major_version 1 it would give an exception in the core imports (bot.Guilds isn't defined)!

### CODE REVIEW

**Comment by Kowlin** [comment]
Hi there, I've changed the base branch to ``V3/develop``, as this is where all our new code ends up before being pushed to ``V3/master``

**Comment by tekulvw** [comment]
Could you post the traceback that occurs without this patch? I'm curious as to what's causing this problem and I'd like to fix it there if at all possible.

**Comment by jorivanee** [comment]
https://hastebin.com/isufotegeh.py @tekulvw, this is fixed in my commit or by updating discord.py to rewrite

**Comment by palmtree5** [comment]
I'm able to repro that when attempting to run v3 with discord.py version 0.16.10.

File: main.py [hunk 1]
```diff
@@ -1,4 +1,12 @@
-from pathlib import Path
+# Discord Version check
+
+import discord
+
+if discord.version_info.major < 1:
+    print("You are not running the rewritten version of discord.py.\n\n"
+          "In order to use Red v3 you MUST be running d.py version"
+          " >= 1.0.0.")
+    sys.exit(1)
```
Comment (palmtree5) [inline main.py:10]: Either:
- `exit(1)` instead of `sys.exit(1)`
OR
- make sure to `import sys` before trying to call `sys.exit`

**Review by palmtree5 — DISMISSED** [review]

**Review by palmtree5 — APPROVED** [review]
Tested, working now

**Comment by palmtree5** [comment]
Also can confirm this is working as a fix

**Review by tekulvw — APPROVED** [review]
LGTM

**Comment by jorivanee** [comment]
Alright thanks.

**Pull request merged**